basin3d/plugins/usgs.py

```diff
@@ -280,6 +285,30 @@ def _load_point_obj(datasource, json_obj, feature_observed_properties, synthesis
         return monitoring_feature
 
 
+def parse_sites_response(usgs_site_response):
+    """
+    Get a dictionary of location variables for the given location results and
+    get a dictionary of unique sites or subbasins
+    :param usgs_site_response: datasource JSON object of the locations
+    :return:
+    """
+    observed_properties_variables = {}
+    unique_usgs_sites = {}
+
+    for v in iter_rdb_to_json(usgs_site_response.text):
+        param, site, stat = v['parm_cd'], v['site_no'], v['stat_cd']
+        observed_properties_variables.setdefault(site, [])
+
+        if param not in observed_properties_variables[site] and map_statistic_code(stat):
```

Comment:
`map_statistic_code(stat)` will always return a value. "NOT_SUPPORTED" is returned if there is no match.

I'm guessing that this is the check to filter by the stats that are supported by the plugin. If so, this should be changed.